### DISCUSSION TITLE
Viewing SlingBox Stream via VLC on Firestick

### DISCUSSION THREAD

**Discussion by ghost** [GerryDazoo/Slinger#57] [General]
Let me just start off by saying - THANK YOU! I got that email from Sling the other day saying they are taking their servers down November 9th and I honestly forgot all about it. I have had a nice setup using my Slingbox since 2015 when i first got Verizon Fios. Because of it, I never had to rent or buy any cable boxes or cards for my other TVs. Just the main one. Using Firesticks or Android boxes with the Slingplayer app has been my solution.

So i tried this Slinger server (loaded it on my main Windows 11 tower) and it worked like a champ. I ended up installing MX Player on my 3 TVs with FireSticks though because the audio is out of sync (by a few seconds - very noticeable) when you use VLC on Firestick with the Slinger stream. It happened on all 3 of my Firesticks too with VLC so definitely not a one-off. MX  Player worked on the first attempt so i just ended up using it instead. 

I do like the VLC GUI a little better because you can make a shortcut right on the main page of the app for the Sling stream. You can't seem to do that for MX player. So basically its just a few extra clicks of the remote to bring up the stream perfectly full screen and in HD on my TVs :) 

Again though, this solution is amazing and it will allow me to keep using my current setup! 

 Prior to discovering this, i was researching other alternatives and nothing seems to do what the Slingbox does at least not in the same exact way with a cable box. I think a lot of other people like me must have gotten that email from Sling the other day...because Slingboxes went way down on eBay! LOL :) I just grabbed one for $20 from a 100% feedbacked seller FYI in case anyone wants a backup box

**Comment by GerryDazoo** [comment]
I've had issues with VLC on android with audio sync but for me MX play works with no issues.

**Reply by GerryDazoo** [reply]
PS. you can go into setting and adjust the audio sync but you have to do it after every restart.

**Comment by JLBla** [comment]
I was wondering how to connect the Firesticks with this Slinger server.  I have set it up using the Windows executable file on my Windows 11 laptop, but didn’t know how to get it to work with the Firesticks and how to install the remote too.  Thanks

**Reply by GerryDazoo** [reply]
you install a media player like VLC or MX player on your firestick and then open a network stream to you Win11 laptop. Make sure your laptop firewall allows connections to your server port number.
